superset-frontend/package.json

```diff
@@ -168,7 +169,8 @@
     "rison": "^0.1.1",
     "shortid": "^2.2.6",
     "urijs": "^1.18.10",
-    "use-query-params": "^0.4.5"
+    "use-query-params": "^0.4.5",
+    "zlib": "^1.0.5"
```

Comment:
why do we need to add this to the deps? I don't see it used anywhere?

Reply:
These path and zlib dependencies keep cropping up for people, and eslint throws an error about them not being in the dependencies. I'm not sure the root cause... resetting the pacakge.json file to what's on master often resolves it, but the issue keeps cropping up like a weed.

Reply:
It is used in `webpack.proxy-config.js` and it triggers `import/no-extraneous-dependencies` rule if it's not listed in deps

Reply:
Same thing here, wondering if it makes sense to move this to `devDependencies` and/or add a `// eslint-disable-next-line import/no-extraneous-dependencies` comment to quiet it.

Reply:
Please use `overrides` to add `webpack` specific configs. E.g.:

```
  overrides: [
    {
      files: 'webpack*.js',
      env: 'node',
      rules: {
        'import/no-extraneous-dependencies': 0,
      },
    },
  ],
```

Reply:
I followed @ktmud's suggestion

Reply:
Awesome! Thanks for jumping in @ktmud